browser/mac/sparkle_glue.mm

```diff
@@ -46,7 +46,7 @@ static void PostPerform(id target, SEL sel, id arg) {
 
     scoped_refptr<PerformBridge> op = new PerformBridge(target, sel, arg);
     base::PostTaskWithTraits(FROM_HERE,
-                             {base::MayBlock(), base::TaskPriority::BACKGROUND,
+                             {base::MayBlock(), base::TaskPriority::BEST_EFFORT,
```

Comment:
are you sure this is equivalent? Sounds worse?

Reply:
`git log -S for BACKGROUND` changed to that in the commit I looked at.